rpcs3/rpcs3qt/pad_settings_dialog.cpp

```diff
@@ -220,7 +220,7 @@ pad_settings_dialog::pad_settings_dialog(std::shared_ptr<gui_settings> gui_setti
 	ChangeProfile(ui->chooseProfile->currentText());
 }
 
-pad_settings_dialog::~pad_settings_dialog()
+void pad_settings_dialog::closeEvent(QCloseEvent* event)
```

Comment:
I don't think it's wise to move cleanup code to a closeEvent.
If you just want to save the geometry, just move the geometry code to the close event.

Reply:
Fixed.

Reply:
?

Reply:
Now fixed, it only signals the thread to abort in closeEvent without joining it, then joining it in the destructor.

Reply:
again... why the change?